README.md

```diff
@@ -44,6 +44,46 @@ By utilizing WarpSQL, you can benefit from the following:
 - *Extensibility*: While WarpSQL already supports a range of extensions, it aims to expand its offerings to include even more powerful tools in the future, such as ZomboDB, PLV8, and Pg Repack.
 Get started with WarpSQL today and experience the convenience of a comprehensive Postgres solution.
 
+## PostgreSQL Image Packer Template
+
+This repository contains a Packer template for building the WarpSQL image with multiple sources and provisioners. The template supports building images based on both the Alpine and Bitnami PostgreSQL images.
+
+#### Prerequisites
+
+Before using this Packer template, ensure that you have the following prerequisites installed:
+
+- [Packer](https://www.packer.io/) 
+- [Docker](https://www.docker.com/) (for building Docker images)
+
+### Usage
+
+To build the WarpSQL image using the Packer template, follow these steps:
+
+1. Clone this repository and navigate to the packer directory:
+
+   ```shell
+    git clone https://github.com/Samagra-Development/WarpSQL.git
+    cd WarpSQL/packer
+2. Build the images:
+    ```shell 
+      packer build warpsql.pkr.hcl
+    ``` 
+To build only the Alpine image, you can use the `-only` option:
+
+  ```shell
+  packer build -only=warpsql.docker.alpine warpsql.pkr.hcl
+  ```
+  By default, all supported extensions are installed. If you want to install specific extensions, you can provide the `extensions` variable:
```

Comment:
Please add a line mentioning the list of extensions they can use at the moment.

Reply:
Sure